card_test/venv/Scripts/code/BLOB.py

```diff
@@ -121,7 +225,7 @@ class Counter:
 
 def generateColors(colorList):  # Generates a list of 100 colors
 
-    for i in range(100):  # A for loop that run 100 times through values of i from 0 to 99
+    for i in range(1000):  # A for loop that run 100 times through values of i from 0 to 99
         # Assigning a random int value between 0 and 255 to each color
```

Comment:
We shouldn't assign random numbers, not critical, but a nice improvement would be to apply unique colors. 

If we have 5 blobs we apply color 20, 40, 60, 80 and 100
if we have 10 we apply 10, 20, 30, 40... You get the idea

Reply:
Also, start naming the branches after the features you are implementing instead of Sara_branch :D